aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -77,6 +77,30 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> Default
 }
 
 impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGroupValue<T, V> {
+    fn set_base_values(
+        &mut self,
+        shifted_idx: ShiftedTxnIndex,
+        incarnation: Incarnation,
+        // TODO add layout to values
+        values: impl IntoIterator<Item = (T, V)>,
+    ) {
+        match self.idx_to_update.get(&shifted_idx) {
+            Some(previous) => {
+                // base value may have already been provided due to a concurrency race.
+                // If maybe_layout is None, they are required to be identical.
```

Comment:
The comment is wrong / old / stale?

Reply:
it is future looking! 
once we add maybe layout (TODO above), that is what check should be doing :D